builtin/providers/azure/provider_test.go

```diff
@@ -11,6 +11,18 @@ import (
 var testAccProviders map[string]terraform.ResourceProvider
 var testAccProvider *schema.Provider
 
+const testAccSecurityGroupName = "terraform-security-group"
+
+// testAccStorageServiceName is used as the name for the Storage Service
+// created in all storage-related tests.
+// It is much more convenient to provide a Storage Service which
+// has been created beforehand as the creation of one takes a lot
+// and would greatly impede the multitude of tests which rely on one.
+// NOTE: the storage container should be located in `West US`.
+var testAccStorageServiceName = os.Getenv("AZURE_STORAGE")
```

Comment:
What's the behavior when this is not set.

In other places we've:

 * Had the provider's setup fail fast with a message when it's not sert.
 * Had the tests that rely on the value being set either skip or fail fast.

From what I can tell the value just ends up empty right now, would be nice to do something more explicit here.